PlayerControls/sources/ContentControlsViewController.swift

```diff
@@ -15,7 +16,7 @@ open class ContentControlsViewController: UIViewController {
         }
     }
     
-    public enum Props: Prism {
+    public enum Props: Prism, AutoCodable {
```

Comment:
You can inherit these two protocols and create a new one

Reply:
@invalid404 We've discussed this case already) Please, look the previous comment)